Sources/Commands/SwiftTestTool.swift

```diff
@@ -220,10 +225,13 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                     toolchain: toolchain,
                     diagnostics: diagnostics,
                     options: self.options,
-                    buildParameters: try self.buildParameters()
+                    buildParameters: buildParameters
                 )
                 ranSuccessfully = runner.test()
 
+                if options.shouldEnableCodeCoverage {
```

Comment:
This should be after this switch case and `executionStatus` assignment. Otherwise, we won't process code coverage data for the other cases.